packages/manager/modules/vps/src/dashboard/vps-dashboard.controller.js

```diff
@@ -47,6 +49,8 @@ export default class {
     this.VpsUpgradeService = VpsUpgradeService;
     this.DASHBOARD_FEATURES = DASHBOARD_FEATURES;
     this.SERVICE_TYPE = SERVICE_TYPE;
+    this.RANGES = RANGES;
+    this.VPS_RANGE_COMPARE_LINKS = VPS_RANGE_COMPARE_LINKS;
```

Comment:
**nitpick:** It is used only in controller file and not in template. So there is no need to attach it to controller object.
```suggestion
```